consumer/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist
```

Comment:
.gitignore works in all subdirectories by default, isn't it?
In any case, you can divide the project into submodules
https://github.blog/2016-02-01-working-with-submodules/